### ISSUE TITLE
Building cmasher

### ISSUE THREAD

**Issue by sinclair-m** [1313e/CMasher#167]
```
___________________________________________ Test_import_cmaps.test_CMasher_cmaps[__pycache__] ___________________________________________

self = <tests.test_utils.Test_import_cmaps object at 0x7f222279a900>, cm_name = '__pycache__'

    @pytest.mark.parametrize("cm_name", cm_names)
    def test_CMasher_cmaps(self, cm_name):
        # Check if provided cm_name is registered in CMasher and MPL
        for name in (cm_name, cm_name + "_r"):
>           cmr_cmap = getattr(cmr, name)
E           AttributeError: module 'cmasher' has no attribute '__pycache__'. Did you mean: '__cached__'?

/home/USERNAME/.cache/yay/python-cmasher/src/CMasher-1.9.0/tests/test_utils.py:358: AttributeError
======================================================== short test summary info ========================================================
FAILED tests/test_utils.py::Test_import_cmaps::test_CMasher_cmaps[__pycache__] - AttributeError: module 'cmasher' has no attribute '__pycache__'. Did you mean: '__cached__'?
```

**Comment by neutrinoceros** [comment]
Can you give me bit of context ? What exactly are you trying to do and what commands are you running ? Thanks !

**Comment by sinclair-m** [comment]
Hello Clément,

For the arch linux installation of cmasher, it runs through the test-env checks of the colormaps https://aur.archlinux.org/packages/python-cmasher

Specifically:

```
check() {
  cd ${_base}-${pkgver}
  python -m venv --system-site-packages test-env
  test-env/bin/python -m installer dist/*.whl
  test-env/bin/python -m pytest
}
```

seems to have an issue with the one check that runs as __pycache__


Best,
Mitchell

**Comment by neutrinoceros** [comment]
Oh I think you hit the very same bug I fixed in https://github.com/1313e/CMasher/pull/101 (the fix was first released in cmasher 1.7.1), which I guess explains why I'm not able to reproduce now. Could you clarify what version of cmasher you're encountering this with ?

**Comment by sinclair-m** [comment]
It looks like it pulling the current github repo:

`HEAD is now at 7831bca Bump version to 1.9.0`

**Comment by neutrinoceros** [comment]
My mistake, it seems I actually broke the logic in #101. I can indeed reproduce now, and I'll publish the fix as part of CMasher 1.9.1
Thank you for reporting !

**Comment by sinclair-m** [comment]
Thank you very much, appreciate the help and the colormaps of course.
